MobileCryptoService/MobileCryptoService/Presentation/HomeScreen/View/ProfileView.swift

```diff
@@ -42,5 +42,6 @@ struct ProfileView: View {
 struct ProfileView_Previews: PreviewProvider {
     static var previews: some View {
         ProfileView()
+        ProfileView().preferredColorScheme(.dark)
```

Comment:
@LeonelAbatteEndava Please review if  .preferredColorScheme(.dark) is ok here please ? I think only use ProfileView() direactly

Reply:
@LeonelAbatteEndava i didn't see that is into the Preview. Sorry we don't have any issue here :) approved